### PR TITLE
Added functionality for external signals

### PR DESCRIPTION
**Pull request by ShogunMan**
Created subfolder signals
Any file with extention .exs will be picked up and parsed
It should be a text file with a list of pairs to be traded.
The file will be removed after it has been picked up

### CODE REVIEW

**Comment by MikkMakk** [comment]
So are you running a second script to obtain Signals and then save them into the Signals Directory and This Bot buys these Pairs from the exs file?

**Comment by ShogunMan** [comment]
Yes, thats how it works.

**Comment by ShogunMan** [comment]
Added sample singnals module. It has its own tickerlist.
Just run from same directory as main program.

File: Binance Detect Moonings.py [hunk 1]
```diff
@@ -118,7 +128,7 @@ def wait_for_price():
         max_price = max(historical_prices, key = lambda x: -1 if x is None else float(x[coin]['price']))
 
         threshold_check = (-1.0 if min_price[coin]['time'] > max_price[coin]['time'] else 1.0) * (float(max_price[coin]['price']) - float(min_price[coin]['price'])) / float(min_price[coin]['price']) * 100
-
+
```
Comment (levdavid) [inline Binance Detect Moonings.py:131]: nit: remove trailing spaces

File: config.yml [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-  TEST_MODE: True
+  TEST_MODE: False
   LOG_TRADES: True
   LOG_FILE: 'trades.txt'
 
 
 # These options apply to the trading methods the script executes
 trading_options:
-  # select what to pair the coins to and pull all coins paired with PAIR_WITH
+  # select what to pair the coins to and pull all coins paied with PAIR_WITH
```
Comment (levdavid) [inline config.yml:12]: typo

File: config.yml [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
   PAIR_WITH: USDT
 
-  # Total amount per trade
+  # Total ammount per trade
```
Comment (levdavid) [inline config.yml:15]: typo

File: config.yml [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
   QUANTITY: 15
 
-  # List of pairs to exclude
+  # List of pairs to exlcude
```
Comment (levdavid) [inline config.yml:18]: typo

**Comment by mindflvx** [comment]
This is great, thanks.

Is there an easy way to choose which indicators to use?

**Comment by ShogunMan** [comment]
Yes, write your own custom signalling module.

File: config.yml [hunk 5]
```diff
@@ -1,21 +1,21 @@
 # These options apply to how the script will operate.
 script_options:
-  # Switch between running in test mode (no buying/selling) and live mode
+  # Switch between testnet and mainnet
   # Setting this to False will use REAL funds, use at your own risk
-  TEST_MODE: True
```
Comment (CyberPunkMetalHead) [inline config.yml:5]: Should probably have TEST_MODE = True as default config

Comment (ShogunMan) [reply config.yml:5]: Sure, seems logical based on the current userbase :-)

**Comment by divs1210** [comment]
The bot has stopped making any transactions since this was merged.

Anyone else seeing this?

**Comment by divs1210** [comment]
Also, now I have to add pairs myself?

I thought the point of the bot was to detect which pair is mooning and buy it?

**Comment by ShogunMan** [comment]
For the %increase yes, for external signalling: not with the current samplemodule.

**Pull request merged**